Lib/test/test_socketserver.py

```diff
@@ -232,12 +233,14 @@ def test_ForkingUDPServer(self):
                             self.dgram_examine)
 
     @requires_unix_sockets
+    @unittest.skipIf(sys.platform == 'win32', "no datagram support")
```

Comment:
I'm seeing a lot of these - is there any more specific way to detect datagram support? The socket module already has a number of conditional constants, so maybe one of those will help?